ride-service/src/main/kotlin/com/dulfinne/taxi/rideservice/dto/request/LocationRequest.kt

```diff
@@ -1,12 +1,16 @@
 package com.dulfinne.taxi.rideservice.dto.request
 
+import com.dulfinne.taxi.rideservice.util.ValidationKeys
+import jakarta.validation.Valid
 import jakarta.validation.constraints.NotNull
 
 data class LocationRequest (
 
-    @field:NotNull(message = "Start position can't be empty")
-    val startPosition: PointRequest,
+    @field:NotNull(message = ValidationKeys.START_POSITION_REQUIRED)
+    @field:Valid
+    val startPosition: PointRequest?,
```

Comment:
My point was to leave not null type, but to remove NotNull annotation because you will need to use (!!) operator to cast the value to not nullable type futher